metrics/psutils/cpu.lua

```diff
@@ -39,8 +44,25 @@ local function update_cpu_metrics()
 
         threads[thread_info.pid] = nil
         new_threads[thread_info.pid] = labels
+
+        instance_cpu_usage.system = instance_cpu_usage.system + thread_info.stime
+        instance_cpu_usage.user = instance_cpu_usage.user + thread_info.utime
     end
 
+    local cpu_label = {}
+
+    local instance_stime_labels = table.copy(cpu_label)
+    instance_stime_labels.kind = 'system'
+    collectors_list.instance_cpu_usage = utils.set_gauge('cpu_instance',
+        'Tarantool instance system CPU time', instance_cpu_usage.system,
+        instance_stime_labels, nil, {default = true})
+
+    local instance_utime_labels = table.copy(cpu_label)
+    instance_utime_labels.kind = 'user'
+    collectors_list.instance_cpu_usage = utils.set_gauge('cpu_instance',
+        'Tarantool instance user CPU time', instance_cpu_usage.user,
```

Comment:
The description belongs to a collector. You cannot set two different descriptions for the same collector with different observation labels.
```suggestion
        'Tarantool instance cpu time', instance_cpu_usage.user,
```

Reply:
Updated.